modules/standard/UtilReplicatedVar.chpl

```diff
@@ -115,10 +144,18 @@ proc rcCollect(replicatedVar: [?D] ?MYTYPE, collected: [?CD] MYTYPE): void
       col = replicatedVar[rcDomainIx];
 }
 
+/*
+Access the copy of `replicatedVar` on the current locale.
+
+This is equivalent to ``replicatedVar[1]``.
+*/
 proc rcLocal(replicatedVar: [?D] ?MYTYPE) ref: MYTYPE {
   return replicatedVar[rcDomainIx];
 }
 
+/*
+Access the copy of `replicatedVar` on the locale `remoteLoc`.
```

Comment:
I feel like a little more description of `replicatedVar` would be useful, but the description of the locale is probably fine.